routers/web/repo/pull.go

```diff
@@ -965,6 +965,11 @@ func MergePullRequest(ctx *context.Context) {
 	}
 
 	log.Trace("Pull request merged: %d", pr.ID)
+
+	if form.DeleteBranchAfterMerge {
+		cleanUpPullRequest(ctx, false)
```

Comment:
You already have the pull request, and git repo open which means that at least until 1204 is already done.

Reply:
I'll see if I can do some cleanup to avoid these unnecessary operations.
How can I access the already opened repo of the PR head?